src/bench/ss_bench/main.rs

```diff
@@ -68,6 +68,18 @@ pub(crate) struct Opts {
     #[clap(long)]
     write_conflict_detection_enabled: bool,
 
+    #[clap(long)]
```

Comment:
Is anyone still using `ss_bench`? If not, I'd suggest turning it into maintenance mode (not compile it by default, and refactor it when someone wants to use it)

Reply:
Maybe in another separate PR.